### PR TITLE
Chapter 1 - Added PHP7 type declarations and added comments to methods  &variables

### PR DESCRIPTION
**Pull request by LoganConnor44**

### CODE REVIEW

**Comment by LoganConnor44** [comment]
It looks as if this continuous integration check failed due to only testing php4-5.

This would be expected because the type declarations used were introduced in 7. I'm going to assume the only edit that needs to take place is adding 7.0 to the travis.yml file.

File: tests/TDD/Chapter1/Chapter1Test.php [hunk 1]
```diff
@@ -2,9 +2,13 @@
...
 namespace TDD\Chapter1;
 
-class Chapter1Test extends \PHPUnit_Framework_TestCase
+class Chapter1Test extends TestCase 
 {
-    public function testMultiplication()
+
+	/**
+	 * Tests that Dollar's public variable, $amount, is the same as the given amount
+	 */
```
Comment (guhelski) [inline tests/TDD/Chapter1/Chapter1Test.php]: There's some extra space between the start of the function declaration and the comments.

**Comment by guhelski** [comment]
Thanks for the changes @LoganConnor44! 👍  I've spotted just one minor formatting issue, I also updated the PHP versions to test against in Travis, so if you rebase this with ```master``` the tests should be green.

**Comment by LoganConnor44** [comment]
- Strange spacing corrected

- Travis.yml pulled and rebased

- Tests are green

**Pull request merged**